app/assets/javascripts/details/state/edit_mode.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+var EditStateController = function (el) {
```

Comment:
@nickbristow Not now. Since these PRs are coming in piecemeal, the "glue" is missing